test/functional/plugin/health_spec.lua

```diff
@@ -223,7 +233,7 @@ describe('health.vim', function()
       -- luacheck: ignore 613
       helpers.expect([[
 
-        non_existent_healthcheck: 
+        non_existent_healthcheck:
```

Comment:
This line needed changing?

Reply:
Apologies but I cannot seem to figure out what happened.  I only made the one change.

Reply:
Looks like a whitespace change, removed whitespace, changed tab for space or vice versa.

Reply:
Thank you.  Now that you said it, exactly what happened is clear.  Somewhat ironic, I had to use `nano` to add-in the space.  My `:w` comes baked-in with removing dangling spaces.